app/portainer/views/devices/profiles/editProfileController.js

```diff
@@ -0,0 +1,84 @@
+import angular from 'angular';
+import { getProfile, updateProfile } from 'Portainer/hostmanagement/fdo/fdo.service';
+
+angular.module('portainer.app').controller('EditProfileController', function ($scope, $async, $state, $window, ModalService, Authentication, Notifications) {
+  $scope.formValues = {
+    Name: '',
+    ProfileFileContent: '',
+  };
+
+  $scope.state = {
+    ProfileID: $state.params.id,
```

Comment:
is there a specific reason why we use a mix of PascalCase and camelCase?

Reply:
not really, mistake, changed it